README.md

```diff
@@ -46,6 +46,10 @@ tfs-server       LoadBalancer   xxxxxxxxxx     xxxxxxxxxx      8500:30869/TCP,85
 kubernetes       ClusterIP      xxxxxxxxxx     <none>          443/TCP                         160m
 ```
 
+## How to perform gRPC inference 
+
+If you wonder how to perform gRPC inference, please check [grpc_client.py](https://github.com/deep-diver/ml-deployment-k8s-tfserving/tree/main/client_grpc_client.py) out. [TFServing API](https://github.com/tensorflow/serving/tree/master/tensorflow_serving/example) provides handy features to construct protobuf request message via `predict_pb2.PredictRequest()`, and `tf.make_tensor_proto(image)` creates protobuf compatible values from `Tensor` data type.
+
```

Comment:
[`grpc_client.py`](https://github.com/deep-diver/ml-deployment-k8s-tfserving/tree/main/client_grpc_client.py) provides code to perform inference with the gRPC client.

Reply:
thanks!